README.md

```diff
@@ -188,15 +188,19 @@ kind: ConfigMap
 [...]
 ```
 - Generate an account token: `argocd account generate-token --account arlon`
-- Make a temporary copy of the config file: `cp ${HOME}/.config/argocd/config /tmp/argocd_config` then
+- Make a temporary copy of the config file: `cp ${HOME}/.config/argocd/config /tmp/config` then
   edit it to replace the value of `auth-token` with the token from
   the previous step. Save changes. This file will be used to configure the Arlon
   controller's ArgoCD credentials during the next steps.
 
+NOTE: _On some operating systems, including Linux, it's possible the source configuration
+file is located at `${HOME}/.argocd/config` instead. In any case, ensure that
+the destionation file is named `/tmp/config`, it's important for the secret creation step below_.
```

Comment:
I don't think it makes a difference whether the temporary copy of the config is `/tmp/config` or `/tmp/argocd_config`. 
However, noted that the path of the ArgoCD config file can be different on MacOS vs Linux (not sure if it has changed between ArgoCD versions). We could also have a script do this by checking whether the OS is MacOS or Linux but not sure if it's consistent across different Linuxes and  ArgoCD versions. We can ask the user to check if the file exists in one of the two locations.

Reply:
Actually the name of the destination file is important, because that's how the data key will be named in the secret. When mounted into the controller pod, the file name HAS to be /.argocd/config, not /.argocd/argocd_config, or else the controller will crash.

Reply:
Saw that /.argocd/config is the default path that ArgoCD looks for a config file unless the path is explicitly specified.
Found that it's not OS specific but ArgoCD has recently tweaked the local path now to use XDG base directory specification to prefer $HOME/.config/.argocd/config over the legacy path $HOME/.argocd/config 

https://github.com/argoproj/argo-cd/blob/master/util/localconfig/localconfig.go#L266